apify-api/openapi/paths/actors/acts.yaml

```diff
@@ -58,6 +61,17 @@ get:
       schema:
         type: boolean
         example: true
+    - name: sortBy
+      in: query
+      description: |
+        Field to sort the records by. The default is `createdAt`. You can also use `lastRunStartedAt` to sort
+        by the most recently ran Actors.
+      style: form
+      explode: true
```

Comment:
Are these intentional? I believe the `sortBy` query parameter should be just a single value.